dashboard/app/views/levels/_teacher.html.haml

```diff
@@ -17,7 +17,10 @@
         - if @user_level
           .user_level
             .level
-              - status = activity_css_class(@user_level.best_result)
+              - if @user_level.submitted?
+                - status = "submitted"
+              - else
+                - status = activity_css_class(@user_level.best_result)
```

Comment:
To reduce churn I will keep it in users_helper, but unifying the best_result & submitted logic in the simple activity_css_class function there.  Reasonable?